content/pt-br/docs/concepts/cluster-administration/manage-deployment.md

```diff
@@ -0,0 +1,460 @@
+
+`kubectl apply` também aceita mútiplos `-f` como argumento:
+
+```shell
+kubectl apply -f https://k8s.io/examples/application/nginx/nginx-svc.yaml -f https://k8s.io/examples/application/nginx/nginx-deployment.yaml
+```
+
+E um diretório pode ser especificado em vez ou além de arquivos individuais:
+
+```shell
+kubectl apply -f https://k8s.io/examples/application/nginx/
+```
+
+`kubectl` lerá qualquer arquivo com sufixos `.yaml`, `.yml`, or `.json`.
+
+É uma prática recomendada colocar recursos relacionados ao mesmo microsserviço ou camada de aplicação no mesmo arquivo, e agrupar todos os arquivos associados com sua aplicação no mesmo diretório. Se as camadas da sua aplicação comunicarem-se entre si usando DNS, você pode realizar o _deploy_ de todos os componentes da sua _stack_ juntos.
+
+Uma URL também pode ser utilizada como fonte de configuração, o que é útil para realizar o _deploy_ diretamente de arquivos de configurações encontrados no GitHub:
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/website/main/content/en/examples/application/nginx/nginx-deployment.yaml
+```
+
+```shell
+deployment.apps/my-nginx created
+```
+
+## Operações em massa no kubectl
+
+A criação de recursos não é a única operação em massa que o `kubectl` pode realizar. Ele também pode extrair nomes de recursos a partir de arquivos de configurações para realizar outras operações, em particular para excluir os mesmos recursos que você criou:
+
+```shell
+kubectl delete -f https://k8s.io/examples/application/nginx-app.yaml
+```
+
+```shell
+deployment.apps "my-nginx" deleted
+service "my-nginx-svc" deleted
+```
+
+No caso de dois recursos, você pode especificar ambos na linha de comando usando a sintaxe _resource/name_:
+
+```shell
+kubectl delete deployments/my-nginx services/my-nginx-svc
+```
+
+Para um maior número de recursos, será mais fácil especificar o seletor (_label query_) especificado usando `-l` ou `--selector`, para filtrar os recursos pelas suas _labels_:
+
+```shell
+kubectl delete deployment,services -l app=nginx
+```
+
+```shell
+deployment.apps "my-nginx" deleted
+service "my-nginx-svc" deleted
+```
+
+Como o `kubectl` gera nomes de recursos na sintaxe aceita, você pode encadear operações usando `$()` ou `xargs`:
+
+```shell
+kubectl get $(kubectl create -f docs/concepts/cluster-administration/nginx/ -o name | grep service)
+kubectl create -f docs/concepts/cluster-administration/nginx/ -o name | grep service | xargs -i kubectl get {}
+```
+
+```shell
+NAME           TYPE           CLUSTER-IP   EXTERNAL-IP   PORT(S)      AGE
+my-nginx-svc   LoadBalancer   10.0.0.208   <pending>     80/TCP       0s
+```
+
+Com os comandos acima, primeiro criamos recursos em `examples/application/nginx/exibimos os recursos criados com `-o name`com o formato de saída (imprimindo cada recurso no formato _resource/name_). Em seguida filtramos apenas o recurso _Service_, utilizando a ferramenta `grep`, e então o imprimimos com `kubectl get`.
```

Comment:
Minor typo: this is showing up with the regular text as preformatted. It is missing a backtick after `nginx/`.